src/camp/generators/lib/campch.py

```diff
@@ -150,6 +151,12 @@ def as_text(ari:ari.ARI) -> str:
         enc.encode(ari, buf)
         return buf.getvalue()
 
+    def as_timespec(value:datetime) -> str:
```

Comment:
Is this argument really a datetime or is it a timedelta?
They will need to be handled differently on the C side